community/index/src/main/java/org/neo4j/index/gbptree/GenSafePointer.java

```diff
@@ -47,6 +47,8 @@
     // unsigned int
```

Comment:
sure, it's absolutely out of scope of this pr and if you have a note about that, it covers this comment